src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -66,13 +65,6 @@ internal sealed class TextBufferIssueTracker : IIssueTracker, IIssueConsumer
         public TextBufferIssueTracker(DTE dte, TaggerProvider provider, ITextDocument document,
             IEnumerable<AnalysisLanguage> detectedLanguages, IIssuesFilter issuesFilter,
             ISonarErrorListDataSource sonarErrorDataSource, ILogger logger)
-            : this(dte, provider, document, detectedLanguages, issuesFilter, sonarErrorDataSource, logger, new IssueToIssueMarkerConverter())
```

Comment:
Now the issue conversion/decoration is handled by the new class, -> the converter parameter is not required -> the internal constructor is not required.